packages/ts/common/numbers.ts

```diff
@@ -115,7 +115,7 @@ export class BigInt extends Uint8Array {
   }
 
   toHexString(): string {
-    return typeConversion.bigIntToHex(this);
+    return "0x" + typeConversion.bigIntToHex(this);
```

Comment:
This will be a major breaking change, while I kind of agree that we could have both I don't think we can merge that.

If AssemblyScript accepts default parameter, maybe we could think of `toHexString(format)` where default would be the same as today keeping compatibility and the format could be plain or prefixed or something along this.

Reply:
Thanks for the review and feedback! I've updated the change with an optional `prefix` that does not affect the compatibility.